curriculum/schema/challenge-schema.js

```diff
@@ -201,6 +218,11 @@ const schema = Joi.object()
       then: Joi.array().items(questionJoi).min(1).required(),
       otherwise: Joi.forbidden()
     }),
+    quizzes: Joi.when('challengeType', {
+      is: challengeTypes.quiz,
+      then: Joi.array().items(quizJoi).min(1).required(),
```

Comment:
We can't make this required as it stands, since the boilerplate quiz md files don't have `--quizzes--` sections.

The quiz component does rely on them, though, so it throws when trying to render. Maybe the solution is to add placeholder questions to all the boilerplates. It's a trivial search and replace job, so I think that's the way to go. What do you reckon?